scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -738,7 +746,7 @@ Remember, you need to use the Ensembl gene identifier to refer to each gene.
 # now do some exploration of other genes on your own! 
```

Comment:
I can't comment on this specifically, but you missed the headers in this notebook between the beginning of differential expression and the end. So `Perform differential expression with DESeq2`, `Create the DESeq object`, and `Exploring the identified differentially expressed genes`.

Reply:
I think those were all fine!

Reply:
Unless I should code quote `DESeq` etc.

Reply:
Or is it because I said title case when I meant sentence case (sigh)

Reply:
> Or is it because I said title case when I meant sentence case (sigh)

It's this one and because I can't properly read the diff that you were doing title -> sentence and not the reverse.